src/OpenTok/OpenTok.php

```diff
@@ -478,6 +478,58 @@ public function forceDisconnect($sessionId, $connectionId)
         return $this->client->forceDisconnect($sessionId, $connectionId);
     }
 
+    /**
+     * Mute a stream in an OpenTok session
+     *
+     * @param string $sessionId The OpenTok session ID that the client is connected to
+     *
+     * @param string $streamId The OpenTok streamId to be muted in the session
+     *
+     * @return bool
+     */
+    public function muteStreamInSession(string $sessionId, string $streamId): bool
+    {
+        // @TODO just one example for the need to refactor: these validation methods
+        // should be deprecated because typehinting is taking care of the validation unless
+        // we make the validators do more work eg. regex
+        Validators::validateSessionId($sessionId);
+        Validators::validateStreamId($streamId);
+
+        try {
+            $this->client->muteStreamInSession($sessionId, $streamId);
+        } catch (\Exception $e) {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * Mute multiple streams in an OpenTok session
+     *
+     * @param string $sessionId The OpenTok session ID that the client is connected to.
+     *
+     * @param array<string> $excludeStreams An array of valid OpenTok stream IDs to mute
+     *
+     * @return bool
+     */
+    public function muteStreamsInSession(string $sessionId, array $excludeStreams): bool
```

Comment:
This does not match the proposed API (`OpenTok.forceMuteAll()`. See https://github.com/opentok/api-review/tree/master/mute-all-api.